requirements.txt

```diff
@@ -4,6 +4,6 @@ Flask-Migrate==2.4.0
 psycopg2-binary==2.7.7
 pytest-cov==2.6.1
 pytest-mock==1.10.1
-pyyaml>=4.2b1
+pyyaml==4.2b1
```

Comment:
Please update this to `pyyaml==5.1` and we'll be good to go!

Reply:
Updated requirements.txt to show pyyaml==5.1